README.md

```diff
@@ -21,12 +22,23 @@ The VDK package has its core feature extraction library written in C, and the re
 
 It also requires a number of Python packages:
 
-  - [numpy](http://www.numpy.org/) (>=1.10.4)
-  - [scipy](http://www.scipy.org/) (>=0.17.0)
-  - [matplotlib](http://matplotlib.org/1.3.1/index.html) (>=1.5.1)
-  - [pandas](http://pandas.pydata.org/) (>=0.17.1)
-  - [scikit-learn](http://scikit-learn.org/stable/) (>=0.18)
-  - [h5py](http://www.h5py.org/) (>=2.2.1)
+  - [numpy](http://www.numpy.org/) (>=1.12.0)
+  - [scipy](http://www.scipy.org/) (>=0.17.1)
+  - [matplotlib](http://matplotlib.org/1.3.1/index.html) (>=2.0.0)
+  - [pandas](http://pandas.pydata.org/) (>=0.19.2)
+  - [scikit-learn](http://scikit-learn.org/stable/) (>=0.18.1)
+  - [h5py](http://www.h5py.org/) (>=2.6.0)
+
+You will need to install gfortran, freetype, pkg-config, freetype and hdf5 (these can't be compiled from source here):
+- on OSX:
+    - `brew install gcc freetype pkg-config homebrew/science/hdf5` (`gcc` has the `gfortran` compiler needed for scipy)
+    - `echo "backend: TkAgg" > ~/.matplotlib/matplotlibrc` (TODO: how can matplotlib work on OSX without this? see http://matplotlib.org/faq/usage_faq.html#what-is-a-backend)
+- on linux: `sudo apt-get install -y gfortran libfreetype6-dev libhdf5-dev` (TODO: test this)
```

Comment:
with these, it is guaranteed that we can simply do 'sudo pip install numpy scipy matplotlib pandas'? Is there still a need to ask the user to follow www.scipy.org/install.html?